strats.md

```diff
@@ -349,6 +350,22 @@ In all strats with an `entranceCondition`, the `from` node of the strat must be
 
 Each of these properties is described in more detail below.
 
+### Come In Normally
+
+A `comeInNormally` entrance condition represents the need for Samus to enter the room through this door, with no other particular requirements. It has no properties.
```

Comment:
This looks like a way to ensure that you just entered the room.
I thought it was a condition for checking that Samus is in some generic state that is able to move normally.
Because `canBabySkip` leaves you in a bouncing Morph ball state, except for the insane version, and does need a condition to model that.

Reply:
Makes sense. At some point I would like to propose we make it so that entrance/exit conditions are the only way to cross between rooms. At the moment, there are implicit strats that cross rooms (i.e. the normal way to cross rooms, in cases where no cross-room strat is being used), and it potentially creates a lot of little issues, due to hidden assumptions which may not always be satisfied. By making the entrances/exits all explicit, it can make it easier to get everything right.

For example, except in cases with a spawnAt, logically you could go through a door into a heated room then back into the original room with no requirements; this could be done to reset obstacles for instance, which would be unsound, since in reality you take some heat damage to open the door and go back through.